### DISCUSSION TITLE
Writing Fingerprint in -q mode

### DISCUSSION THREAD

**Discussion by mattheys** [beetbox/beets#3969] [Ideas]
Hi, I noticed that tracks gets ACOUSTID_FINGERPRINT written when importing, since this is irrespective to any internet data or match are there any options I can set that will always write that to the file so it doesn't need to be recalculated every pass? 

I have a lot of old stuff to import so I was doing a first pass in -q mode to get all the low hanging fruit first, then I would go through and do it manually. 

If I can get the fingerprints written to the file in the first pass then I wouldn't need to do it in the second one which would hopefully speed things up a little bit.

**Comment by wisp3rwind** [comment]
I don't think this is currently possible, but in general, remembering the fingerprint seems like a reasonable feature to add. In might be worth discussing whether this should really be written to the original file (which would change the current behavior of not modifying files that don't end up being imported) or whether to store this in a separate state file.

**Reply by mattheys** [reply]
Thanks, and I agree that it probably isn't best to change the default behaviour as people won't be expecting that which could have unintended consiquences e.g. their file modify times change or additional files to get backed up during a backup procces. It might be nice to have an override for those who do wish that and as you say keep the default behaviour to record the data in a file.
